File: js/retrieve-linked-content/GetSimpleArticleWithAuthor.js
// DocSection: linked_content_get_article_with_author
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
const KontentDelivery = require('@kentico/kontent-delivery');

const deliveryClient = KontentDelivery.createDeliveryClient({
  projectId: '8d20758c-d74c-4f59-ae04-ee928c0816b7',
});

const response = await deliveryClient.item('the_origin_of_coffee')
  .depthParameter(1)
  .toPromise();
// EndDocSection

File: ruby/retrieve-linked-content/GetSimpleArticleWithAuthor.rb
# DocSection: linked_content_get_article_with_author
# Tip: Find more about Ruby SDKs at https://kontent.ai/learn/ruby
require 'delivery-sdk-ruby'

delivery_client = Kontent::Ai::Delivery::DeliveryClient.new project_id: '8d20758c-d74c-4f59-ae04-ee928c0816b7'
delivery_client.item('the_origin_of_coffee')
               .depth(1)
               .execute do |response|
                 item = response.item
                 puts response.json
               end
# EndDocSection